resolver/min_conns_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	"context"
 	"testing"
 
-	. "github.com/bufbuild/httplb/resolver" //nolint:revive
+	. "github.com/bufbuild/httplb/resolver" //nolint:revive // use dot import for tests
```

Comment:
Oh, I didn't realize revive just straight-up disallows dot imports.

Maybe that would be better as a wildcard rule for `.*_test\.go` in the YAML config file, so we can just opt out of this rule for tests entirely.